playbooks/infra-prometheus/requirements.yml

```diff
@@ -1,2 +1,4 @@
 - src: https://github.com/redhat-cop/infra-ansible
   version: v1.0.16
+- src: cloudalchemy.node_exporter
```

Comment:
Please document the use of this in a README - i.e.: including a link to the originating implementation, etc. etc. to give proper credit to the source.